### PR TITLE
add echos to check sha256sum

### PR DESCRIPTION
**Pull request by chrismcg14**

### CODE REVIEW

File: scripts/signing-helpers.sh [hunk 1]
```diff
@@ -81,6 +88,12 @@ expect {
     timeout close
 }
 END
+
+    for RPM in $RPMS; do
+        POSTRPMSHA=$(sha256sum $RPM)
+        echo -------Pre sig sha---------
```
Comment (gschueler) [inline scripts/signing-helpers.sh:94]:
```suggestion
        echo -------Post sig sha---------
```

File: scripts/signing-helpers.sh [hunk 2]
```diff
@@ -122,11 +142,23 @@ expect {
     timeout { puts "Timed out!"; exit 1 }
 }
 END
+
+for DEB in $DEBS; do
+        POSTDEBSHA=$(sha256sum $DEB)
+        echo -------Pre sig sha---------
```
Comment (gschueler) [inline scripts/signing-helpers.sh:148]:
```suggestion
        echo -------Post sig sha---------
```

**Comment by gschueler** [comment]
might be useful to echo the filename or full filepath alongside sha

**Comment by chrismcg14** [comment]
@gschueler updated

**Review by gschueler — APPROVED** [review]

**Pull request merged**